extensions/amp-iframe/1.0/component.js

```diff
@@ -75,7 +79,9 @@ export function Iframe({
         return;
       }
       dataRef.current = event.data;
-      attemptResize();
+      if (dataRef.current !== null) {
```

Comment:
This one should be checking for `isIntersecting`
```suggestion
      if (isIntersectingRef.current) {
```

Reply:
Updated this to `isIntersectingRef.current !== null` since `isIntersectingRef.current` can equal `false`.